apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -118,6 +125,19 @@ const FinishStudentAccount: React.FunctionComponent<{
     sessionStorage.setItem(USER_GENDER_SESSION_KEY, newGender);
   };
 
+  const sendFinishEvent = (): void => {
+    analyticsReporter.sendEvent(
+      EVENTS.SIGN_UP_FINISHED_EVENT,
+      {
+        'user type': 'student',
+        'has school': false,
+        'has marketing value selected': true,
+        'has display name': !showNameError,
```

Comment:
For my understanding, is this the standard for how we format event names on Statsig (i.e. with spaces)?

Reply:
I am not sure we have a standard for the optional event params, but was just matching what we have on production for the old flow!